R/prettyCoOncoplot.R

```diff
@@ -23,23 +23,22 @@
 #' @return A ggplot object with 2 oncoplots side-by-side.
 #'
 #' @rawNamespace import(ggpubr, except = "get_legend")
-#' @import ComplexHeatmap dplyr maftools
+#' @import ComplexHeatmap dplyr
 #' @export
 #'
 #' @examples
 #' #get data for plotting
 #' meta = GAMBLR.data::gambl_metadata
```

Comment:
Perhaps it would look nicer if we get the metadata using the GAMBLR function created for this, i.e. not just load the bundled `.Rda` object?

Reply:
![coonco](https://github.com/morinlab/GAMBLR.viz/assets/32147626/d499c7de-6867-4247-b79d-b21f63d09a08)